modulefiles/build.ursa.intelllvm.lua

```diff
@@ -1,16 +1,16 @@
 help([[
-Load environment to compile UFS_UTILS on S4 using Intel
+Load environment to compile UFS_UTILS on Ursa using Intelllvm
 ]])
 
-prepend_path("MODULEPATH", "/data/prod/jedi/spack-stack/spack-stack-1.6.0/envs/upp-addon-env/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/contrib/spack-stack/spack-stack-1.9.1/envs/ue-oneapi-2024.2.1/install/modulefiles/Core")
```

Comment:
Same note on 1.9.2

Reply:
here's the Ursa intelllvm going into UFSWM: https://github.com/ufs-community/ufs-weather-model/pull/2471/files#diff-b1cba59a13bc0b3b43a5a65eea775f08e42552877640dd80e2ded02dd00cea0a